store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorageTest.java

```diff
@@ -443,19 +451,19 @@ public void shouldGetGraphsInOrder() throws Exception {
         final List<String> configBA = Arrays.asList(b.getDeserialisedConfig().getGraphId(), a.getDeserialisedConfig().getGraphId());
 
         // When
-        final Collection<Graph> graphsAB = graphStorage.get(authUser, configAB);
-        final Collection<Graph> graphsBA = graphStorage.get(authUser, configBA);
+        final Collection<GraphSerialisable> graphsAB = graphStorage.get(authUser, configAB);
```

Comment:
GraphSerialisable cannot read the data unlike Graph, so this is faster? Or just an implementation detail?

Reply:
`GraphSerialisable` is what is stored in the Cache. Most of the time information about the graphs is required not the actual graph, so it is quicker and easier to pass the `GraphSerialisable` around and only actually make a graph the 1 time you want to execute an operation.